### PR TITLE
Add another improvement to tribble error code

### PR DESCRIPTION
**Pull request by yfarjoun**
- cleaned up some unneeded files

### Description

Please explain the changes you made here.
Explain the **motivation** for making this change. What existing problem does the pull request solve?

### Things to think about before submitting:
- [ ] Make sure your changes compile and new tests pass locally.
- [ ] Add new tests or update existing ones:
  - A bug fix should include a test that previously would have failed and passes now.
  - New features should come with new tests that exercise and validate the new functionality.
- [ ] Extended the README / documentation, if necessary
- [ ] Check your code style.
- [ ] Write a clear commit title and message
  - The commit message should describe what changed and is targeted at htsjdk developers
  - Breaking changes should be mentioned in the commit message.

### CODE REVIEW

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java [hunk 1]
```diff
@@ -418,9 +418,9 @@ public WFIterator iterator() {
      * Iterator for a query interval
      */
     class QueryIterator implements CloseableTribbleIterator<T> {
-        private String chrAlias;
-        int start;
-        int end;
+        String chrAlias;
```
Comment (lbergelson) [inline src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: we lost a private here for some reason?

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java [hunk 2]
```diff
@@ -429,6 +429,8 @@ public WFIterator iterator() {
         public QueryIterator(final String chr, final int start, final int end, final List<Block> blocks) throws IOException {
             this.start = start;
             this.end = end;
+            // for error message
+            chrAlias = chr;
```
Comment (lbergelson) [inline src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: this is already set below and should probably be final

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java [hunk 3]
```diff
@@ -461,7 +462,7 @@ private void advanceBlock() throws IOException {
             while (blockIterator != null && blockIterator.hasNext()) {
                 final Block block = blockIterator.next();
                 if (block.getSize() > 0) {
-                    final int bufferSize = Math.min(2000000, block.getSize() > 100000000 ? 10000000 : (int) block.getSize());
+                    final int bufferSize = Math.min(2_000_000, block.getSize() > 100_000_000 ? 10_000_000 : (int) block.getSize());
```
Comment (lbergelson) [inline src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: Is this a bug here?  I totally assumed those were the same number.

Comment (lbergelson) [reply src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: On closer inspection, this whole line seems busted.  
It seems like what they meant to write was `Math.min(2_000_000, Math.min(10_000_000, block.getSize()))`, but then typoed the number of 0's.  But I think as it's currently written (or as I think they intended it, it just reduces to `Math.min(2_000_000, block.getSize())` anyway.  

I'm not sure what the correct behavior is, but this seems wrong in any case.

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java (hunk 2 above)
Comment (yfarjoun) [reply src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: I actually need this for the case that an error happens on the first line that is being read....happy to explain more, but this is not a mistake.

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java (hunk 3 above)
Comment (yfarjoun) [reply src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: I just added '_' so it (human)readable....if you want to change the numbers, please use a followup PR.

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java (hunk 2 above)
Comment (lbergelson) [reply src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: Oh.  I see.  Maybe write that into the comment?

File: src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java (hunk 3 above)
Comment (lbergelson) [reply src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java]: I know, it just surprised me that those numbers weren't the same, which made me look at it.

**Comment by yfarjoun** [comment]
better now @lbergelson

**Pull request merged**